nntrainer/graph/network_graph.cpp

```diff
@@ -353,10 +363,15 @@ sharedConstTensors NetworkGraph::forwarding(
   bool training,
   std::function<void(std::shared_ptr<LayerNode>, bool)> forwarding_op,
   std::function<bool(void *userdata)> stop_cb, void *userdata) {
+
+  for (auto w : clip_weights) {
```

Comment:
I wonder if we have to enable gradient clip property also true to use mixed precision training.  I guess, this PR doesn't consider the case which enabled mixed + gradient clip.